src/services/formatting/smartIndenter.ts

```diff
@@ -322,6 +322,26 @@ namespace ts.formatting {
             return false;
         }
 
+        export function childStartsInlineWithPreviousObject(parent: Node, child: TextRangeWithKind, childStartLine: number, sourceFile: SourceFileLike): boolean {
+            if (parent.kind === SyntaxKind.CallExpression) {
+                const parentCallExpression = <CallExpression>parent;
+                const currentNode = find(parentCallExpression.arguments, arg => arg.pos === child.pos);
+                if (!currentNode) return false; // Shouldn't happen

```

Comment:
If this is intended to be a guarantee, and you’d prefer to get a bug report instead of silently not working if the assumption doesn’t hold, you could consider doing `const currentNode = Debug.assertDefined(...)` (which also narrows the type to eliminate `undefined`). But for a boolean-returning helper function, I can see it either way—returning false sounds technically correct as well.